eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -322,12 +322,10 @@ jobs:
     # Generate Documentation
     - pwsh: |
         $setupFile="doxygen-1.8.18-setup.exe";
+        $url = "https://azuresdktooling.blob.core.windows.net/dependencies/$setupFile"
```

Comment:
nit: Can we separate this change out into its own PR and leave the scope of this PR just a targeted fix for the issue?

Reply:
Sure. This didn't improve perf as much as hoped either. `Invoke-WebRequest` can get slow and unreliable when working with large files and I've been trying to eliminate it as I come across it.